raiden/tests/unit/rif_comms/test_rif_comms_client.py

```diff
@@ -1,181 +1,145 @@
-            # got this from subscription of lumino node
-        )
-
-        some_raiden_message = {
-            'type': 'LockedTransfer',
-            'chain_id': 33,
-            'message_identifier': 9074731958492744333,
-            'payment_identifier': 2958725218135700941,
-            'payment_hash_invoice': '0x0000000000000000000000000000000000000000000000000000000000000000',
-            'nonce': 45,
-            'token_network_address': '0xd548700d98f32f83b3c88756cf340b7f61877d75',
-            'token': '0xf563b16dc42d9cb6d7ca31793f2d62131d586d05',
-            'channel_identifier': 12,
-            'transferred_amount': 17000000000000000000,
-            'locked_amount': 1000000000000000000,
-            'recipient': '0x00e8249ee607ea67127c4add69291a6c412603c5',
-            'locksroot': '0x043c092c72059c4c154cb342409d95364888a98fa87efadece5add9c255dce9a',
-            'lock': {
-                'type': 'Lock',
-                'amount': 1000000000000000000,
-                'expiration': 1165251,
-                'secrethash': '0x2dc5a7ff26be395d443200db5d16b5d2aadae3a83836be648cd4cba8e2e555fe'
-            },
-            'target': '0x00e8249ee607ea67127c4add69291a6c412603c5',
-            'initiator': '0x27633dc87378a551f09f2fcf43a48fc2b3425d43',
-            'fee': 0,
-            'signature': '0xbaa4df61ab23ab8fdddf4a90fd5db7dd04da364795ebb717426bdd1fcefb411759e1b41d71c3c54b2a75c3eba0f2a2c54c846a0139120ee794f51b0e2a0d954d1c'
-        }
-
-        stub.SendMessageToTopic(
-            PublishPayload(
-                topic=Channel(channelId="16Uiu2HAm9otWzXBcFm7WC2Qufp2h1mpRxK1oox289omHTcKgrpRA"),
-                message=Msg(payload=str.encode(json.dumps(some_raiden_message)))
-            )
-        )
-
-    @pytest.mark.skip(reason="ignore")
-    def test_two_clients_topic_subscription(self):
-        notification_stream_1 = self.rif_comms_client1.connect()
-
-        notification_stream_2 = self.rif_comms_client2.connect()
-
-        one_own_sub = self.rif_comms_client1.subscribe_to(LUMINO_1_ADDRESS)
-        one_two_sub = self.rif_comms_client1.subscribe_to(LUMINO_2_ADDRESS)
-
-        two_own_sub = self.rif_comms_client2.subscribe_to(LUMINO_2_ADDRESS)
-        two_one_sub = self.rif_comms_client2.subscribe_to(LUMINO_1_ADDRESS)
-
-        peer_id_1 = self.rif_comms_client2._get_peer_id(LUMINO_2_ADDRESS)
-
-    @pytest.mark.skip(reason="ignore")
-    def test_two_clients_listen_same_topic(self):
-
-        notification_stream_1 = self.rif_comms_client1.connect()
-
-        notification_stream_2 = self.rif_comms_client2.connect()
-
-        one_own_sub = self.rif_comms_client1.subscribe_to(LUMINO_1_ADDRESS)
-        two_one_sub = self.rif_comms_client2.subscribe_to(LUMINO_1_ADDRESS)
-
-        peer_id_1 = self.rif_comms_client2._get_peer_id(LUMINO_2_ADDRESS)
-
-        for resp in one_own_sub:
-            print("Respone for 1: ", resp)
-
-        for resp in two_one_sub:
-            print("Respone for 2: ", resp)
```

Comment:
this didn't match the new signature for rif comms subscription calls

Reply:
not specifically that line, but the whole test.

for starts, `subscribe_to` now outputs 2 variables instead of one, which didn't match the code.

also, unless a message was sent, these `for` statements would just hang, since the `subscribe_to` code now fetches the first notification which contains the topic ID.